### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by nishchalraj** [atul161/Testgit#1]
finding any view in the actiivity should generally be on <b><i>onCreate</i></b>, these <b>EditText et1=findViewById(R.id.et1);
EditText et2=findViewById(R.id.et2);</b> should be found previously and not on the <b><i>onClick</i></b> of the button (generally) or if I'm wrong please correct me.

**Comment by atul161** [comment]
Yes bhaiya generally we find id of an element in onCreate method..
But but it is more suitable for me if i declare refrence variable of class above and find its id when we have to use that... Yes but generally we find id of an element in onCreate method.
